oncocnv/v7.0/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:20.04 AS download
+LABEL base_image="ubuntu:20.04" \
```

Comment:
labels should be in final image, not intermediate one used to download file

Reply:
done